src/js/_enqueues/wp/customize/controls.js

```diff
@@ -693,6 +693,13 @@
 		construct = this;
 		params = params || {};
 		focus = function () {
+			// If a child section is currently expanded, collapse it.
+			api.section.each( function ( section ) {
```

Comment:
Later on in this function, there are checks to determine whether the current construct has extended `api.Panel`. Because this bug affects only panels, would it be prudent to also check here as well whether the construct is a panel before beginning a loop?

Also, this is looping over all registered sections, but my understanding of the bug was that it affected a panel only when one of its child sections was focused. If that's so, and if we confirm that the construct to be focused is a panel, could we instead loop over just over the current panel's sections to ensure they're collapsed? So, something like:

```
			if ( construct.extended( api.Panel ) ) {
				api.panel( construct.id ).sections().forEach( function ( section ) {
					if ( section.expanded() ) {
						section.collapse();
					}
				} );
			}
```

Reply:
> ```js
> if ( construct.extended( api.Panel ) ) {
> 	construct.sections().forEach( function ( section ) {
> 		if ( section.expanded() ) {
> 			section.collapse();
> 		}
> 	} );
> }
> ```

Thanks @dlh01! I've read both reviews, tested and implemented this rewrite above.